projects/components/src/datagrid/datagrid.component.html

```diff
@@ -111,22 +111,24 @@ <h3 *ngIf="this.header" class="vcd-header">{{ header }}</h3>
             </div>
         </clr-dg-cell>
 
-        <clr-dg-cell *ngFor="let column of columnsConfig">
-            <!-- Default renderer -->
-            <ng-container *ngIf="column.fieldName">{{ restItem | nestedProperty: column.fieldName }}</ng-container>
+        <div *ngFor="let column of columnsConfig">
+            <clr-dg-cell [vcdShowClippedText]="column.cliptextConfig">
+                <!-- Default renderer -->
+                <ng-container *ngIf="column.fieldName">{{ restItem | nestedProperty: column.fieldName }}</ng-container>
 
-            <!-- Renderer is a function -->
-            <ng-container *ngIf="column.fieldRenderer">{{
-                restItem | functionRenderer: column.fieldRenderer
-            }}</ng-container>
+                <!-- Renderer is a function -->
+                <ng-container *ngIf="column.fieldRenderer">{{
+                    restItem | functionRenderer: column.fieldRenderer
+                }}</ng-container>
 
-            <!-- Renderer is a componentRenderer -->
-            <ng-template
-                *ngIf="column.fieldColumnRendererSpec"
-                [vcdComponentRendererOutlet]="{ rendererSpec: column.fieldColumnRendererSpec, context: restItem }"
-            >
-            </ng-template>
-        </clr-dg-cell>
+                <!-- Renderer is a componentRenderer -->
+                <ng-template
+                    *ngIf="column.fieldColumnRendererSpec"
```

Comment:
Please move the `*ngIf` to the same line as `<ng-template>`, I know it's not your doing but that's how we typically do it